src/clj/puppetlabs/services/jruby_pool_manager/jruby_core.clj

```diff
@@ -219,10 +219,10 @@
    event-callbacks :- [IFn]]
   (let [timeout (get-in pool-context [:config :borrow-timeout])
         requested-event (jruby-events/instance-requested event-callbacks reason)
-        instance (pool-protocol/borrow-with-timeout
-                   pool-context
-                   timeout)]
-    (jruby-events/instance-borrowed event-callbacks requested-event instance)
+        [instance worker-id] (pool-protocol/borrow-with-timeout
+                          pool-context
```

Comment:
weird spacing

Reply:
what should this align with?

Reply:
I guess I can make it like how it was previously